src/ai_provider/google_cloud_ai_provider.py

```
@@ -0,0 +1,31 @@
+        file name, and its contents.
+
+        Args:
+            file_name (str): The name of the file to document.
+            project_path (str): The project path where the file is located.
+            file_contents (str): The contents of the file to be documented.
+
+        Returns:
+            str: The generated documentation for the file.
+        """
+
+        prompt = self.generate_prompt(file_name, project_path, file_contents)
```

Ruff

Local variable `prompt` is assigned to but never used

Remove assignment to unused variable `prompt`

(F841)